Macros/e3 Includes/e3_Basics.inc

```diff
@@ -2418,3 +2419,42 @@ SUB check_Forage
 	}
 /if (${Debug}) /echo <== check_Forage -|
 /RETURN
+
+|----------------------------------------------------------------------------|
+|- Sends instance invites to all those in your group or raid.
+|- Install Instructions:
+|-     Add this SUB to the bottom of e3 Includes/e3_Basics.inc
+|-     Add /doevents sendInstanceInvites to SUB basics_Background_Events
+|- Usage Instructions:
+|-     Use "/bc send instance invites" on the toon in the instance
+|-     to send out instance invites.
+|----------------------------------------------------------------------------|
+#event sendInstanceInvites "<#1#> send instance invites"
+SUB event_sendInstanceInvites(line, ChatSender)
+	/if (!${Me.Name.Equal[${ChatSender}]}) /return
+
+	/if (!${Me.InInstance}) {
+		/echo You are not in an instance.
+		/return
+	}
+	
+	/if (${Raid.Members} == 0 && ${Group.Members} == 0) {
+		/echo You are not in a group or raid. Therefore, there is no one to invite.
+	} else /if (${Raid.Members} == 0 && ${Group.Members} > 0) {
+		/echo Sending instance invites to all group members.
+		/declare i int local
+		/for i 1 to ${Group.Members} {
+			/say #inst invite ${Group.Member[${i}]}
+		}
+		/next i
+	} else /if (${Raid.Members} > 0) {
+		/echo Sending instance invites to all raid members.
+		/declare i int local
+		/for i 1 to ${Raid.Members} {
+			/say #inst invite ${Raid.Member[${i}]}
```

Comment:
You've tested this with a lot of raid members? It doesn't get rate limited?

Reply:
If necessary, you could build up a long string of names, then emit the #inst invite ${longStringOfNames}